pubs/plugs/git/git.py

```diff
@@ -0,0 +1,44 @@
+
+import subprocess
+from pipes import quote as shell_quote
+
+from ...plugins import PapersPlugin
+from ...events import *
```

Comment:
Avoid wildcard imports. Always make them explicit (otherwise it is hard to tell what comes from where and you may have namespace collisions).